src/woo_publications/publications/admin.py

```diff
@@ -826,3 +834,17 @@ def show_actions(self, obj: Topic) -> str:
             '<a href="{}">{}</a>',
             actions,
         )
+
+
+@admin.register(PublicationIdentifier)
+class PublicationIdentifierAdmin(AdminAuditLogMixin, admin.ModelAdmin):
+    list_display = (
+        "kenmerk",
+        "bron",
+        "publicatie",
+    )
+    search_fields = (
+        "kenmerk",
+        "bron",
+        "publicatie__uuid",
+    )
```

Comment:
```suggestion
```

I don't see much value in exposing this by itself. However, it might be good to extend the `search_fields` of publication in the admin to also search through the related identifiers `kenmerk` field